src/Http/Models/Invite.php

```diff
@@ -39,6 +39,14 @@ public function __construct(array $attributes = [])
         parent::__construct($attributes);
         $this->setTable(config('invite-codes.tables.invites_table'));
     }
+    
```

Comment:
I think this line has blank spaces. Remove it and add a new line. It should fix the styleci checks.

Reply:
tried but github does not recognize it as a difference

Reply:
Add style ci to your repo. Then, you can analyze it there and merge a pull request in your branch fixing this issue.